charts/rafiki-auth/Chart.yaml

```diff
@@ -17,7 +17,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.4.0
+version: 0.4.1
```

Comment:
There is a version clash with your other PR but I'm sure you know

Reply:
Yes I know. Thank you for pointing that out for me. I don't know which PR will be merged first, so I bumped patch version for both PR. When first one will be merged, the second one will get new patch number.